doc/_static/images/Opeteams_Favicon-01.svg

```diff
@@ -0,0 +1,50 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I can't make it as a suggestion, but could you rename this to `openteams-favicon.svg`? It fixes a typo in the filename, follows the conventions used for the rest of the images (the naming of `spyder_logo`/`_banner` was an oversight when Isabella updated it), and avoids a superfluous suffix.